README.md

```diff
@@ -43,3 +43,7 @@ yarn prepare:avalanche
 ```
 yarn prepare:polygon
 ```
+
+## Subgraph Documentation
+
+Airswap v3 Documentation can be found here -> https://hackmd.io/@6Vt_l5I0TP6t3O_awqTYkA/ryKJ5FIss
```

Comment:
It'd be best to include the link content directly in the README.

Reply:
And to include usage instructions, e.g. scripts like `build` and `codegen`.

Reply:
The usage script are in the readMe.